translations/en-us.yml

```diff
@@ -942,6 +942,7 @@ registries:
     discover:
         page_title: Search
         SHARE: SHARE
+        registrations: Registrations
```

Comment:
Should this be in quotes?

Reply:
Quotes are generally not needed unless this is a multi-word string.

Reply:
Maybe we enforce the rule of always using quotes?

Reply:
This is beyond scope. Prefer we decide together as a team on this. Will make a note of this.